src/ui/components/Search/component.ts

```diff
@@ -13,15 +13,15 @@ export default class Home extends Component {
     isOnline: boolean;
     store: Store;
     searchTerm: string;
-    coordinates: number[];
+    coordinates: string;
     searchResults: string[];
     updateFogEffect: (index: number) => void;
     pullIndexedDB: () => void;
     router: Navigo;
   };
 
   @tracked
-  coordinates: number[] | null;
+  coordinates: string | null;
```

Comment:
Hm, I don't understand why this change is necessary. Could you explain?

Reply:
It turned out that the actual value has been a plain string all this time. It was a mistake to have it as `number[]`.

Reply:
> It turned out that the actual value has been a plain string all this time.

Hm, we should fix that instead then. I guess we'd only have to update the [routing code here](https://github.com/simplabs/breethe-client/blob/master/src/ui/components/Breethe/component.ts#L131).

Reply:
I'm looking into this but I'm not sure parsing and formatting is beneficial for us. The separated values are never used in the app. We'd have to parse it on the function you mentioned and then serialize them as a string again when we send them to the server.

If we decide to split it, I think we should keep the numbers as strings to avoid losing precision on the coordinates with JS' floats.

Reply:
I think we could also treat the coordinates as strings but split them into an array of strings at least. That should be relatively easy by simply modifying the regexp actually I think?